recipes/co/all/conanfile.py

```diff
@@ -14,6 +14,7 @@ class CoConan(ConanFile):
     topics = ("co", "coroutine", "c++11")
     exports_sources = "CMakeLists.txt", "patches/*"
     generators = "cmake", "cmake_find_package"
+    deprecated = "cocoyaxi"
```

Comment:
As I said, this modification should be in another PR than this one. Indeed conan-center-index doesn't allow to modify 2 different recipes in one.